src/main/java/betterquesting/client/gui2/GuiQuestLines.java

```diff
@@ -183,8 +183,12 @@ public void initPanel()
         cvBackground.addPanel(btnExit);
 
         // Search button
+        initSearchPanel();
         PanelButton btnSearch = new PanelButton(new GuiTransform(GuiAlign.BOTTOM_LEFT, 8, -40, 32, 16, 0), -1, "").setIcon(PresetIcon.ICON_ZOOM.getTexture());
         btnSearch.setClickAction(this::openSearch);
```

Comment:
I think this line is redundant with the next now? I think we can also delete `openSearch()` entirely, since this line is the only reference to it